### ISSUE TITLE
Must env_vars be hardcoded into .kitchen.yml?

### ISSUE THREAD

**Issue by j4m3s** [neillturner/kitchen-verifier-serverspec#8]
I'm using option 3: 'Locally on your workstation running serverspec in ssh mode'

At the start of a 'kitchen verify' run, test-kitchen knows the details of the instance it is testing, and outputs them to the terminal, e.g. 
```
[Serverspec] Verify on instance=#<Kitchen::Instance:0x0000000280a2f8> with state={:hostname=>"127.0.0.1", :port=>"2222", :username=>"vagrant", :ssh_key=>
```
Can those connection details not be automatically be picked-up and used for the test suite, rather than having to hard-code them into the .kitchen.yml file as described in the README?:
```yaml
      env_vars:
        TARGET_HOST: 127.0.0.1
        TARGET_PORT: 2222
        LOGIN_USER: vagrant
        SSH_KEY: 'c:/repository/puppet_repo/private_key.pem'
```

**Comment by j4m3s** [comment]
I'm not a ruby man so have resorted to hacking the `serverspec.rb` file in my .gem directory.  I've inserted the following before line 376:
```rb
          ENV['KITCHEN_' + key.to_s.upcase] = value.to_s
          info("Environment variable #{'KITCHEN_' + key.to_s.upcase} value #{value}")
```

Now, back in my kitchen folder I can set my `spec/spec_helper.rb` to contain:
```rb
  set :host, ENV['KITCHEN_HOSTNAME']
  # ssh options at http://net-ssh.github.io/ssh/v1/chapter-2.html
  # ssh via ssh key
  set :ssh_options, :user => ENV['KITCHEN_USERNAME'], :port => ENV['KITCHEN_PORT'], :paranoid => false, :verbose => :error, :host_key => 'ssh-rsa', :keys => [ ENV['KITCHEN_SSH_KEY'] ]
```
and everything works dynamically :)

I'm sure there's a neater way but it works! 

(NB: I think the README needs updating to show the `:port` being set in `spec/spec_helper.rb`?)

**Comment by neillturner** [comment]
that's a good idea i will add it and update the readme.

**Comment by neillturner** [comment]
fixed in 0.5.0
